cloudslang_cli.rst

```diff
@@ -110,16 +110,34 @@ Configure the CLI
 The CLI can be configured using the configuration file found at
 ``cslang/configuration/cslang.properties``.
 
-+---------------------+---------------------------------------------------------+-----------------------+
-| Configuration key   | Default value                                           | Description           |
-+=====================+=========================================================+=======================+
-| log4j.configuration | file:${app.home}/configuration/logging/log4j.properties | | Location of logging |
-|                     |                                                         | | configuration file  |
-+---------------------+---------------------------------------------------------+-----------------------+
-| cslang.encoding     | utf-8                                                   | | Character encoding  |
-|                     |                                                         | | for input values    |
-|                     |                                                         | | and input files     |
-+---------------------+---------------------------------------------------------+-----------------------+
+Some of the configuration items are listed in the table below:
+
++-------------------------------------+---------------------------------------------------------+--------------------------+
+| Configuration key                   | Default value                                           | Description              |
++=====================================+=========================================================+==========================+
+| log4j.configuration                 | file:${app.home}/configuration/logging/log4j.properties | | Location of logging    |
+|                                     |                                                         | | configuration file     |
++-------------------------------------+---------------------------------------------------------+--------------------------+
+| cslang.encoding                     | utf-8                                                   | | Character encoding     |
+|                                     |                                                         | | for input values       |
+|                                     |                                                         | | and input files        |
++-------------------------------------+---------------------------------------------------------+--------------------------+
+| maven.home                          | ${app.home}/maven/apache-maven-3.3.9                    | | Location of CloudSlang |
```

Comment:
maybe we could say here something like `${app.home}/maven/apache-maven-x.y.z` so we  don;t have to update the doc each time we upgrade maveb
WDYT?